bloom-housing/bloom
# feat(expandablecontent): upgrade ExpandableContent to gen 2.0 styling

# Pull Request Template

## Issue Overview

This PR addresses #issue


## Description

Update the styling of the [ExpandableContent ](https://storybook.bloom.exygy.dev/?path=/story/actions-expandable-content--standard) to follow the [gen 2](https://github.com/bloom-housing/bloom/blob/dev/docs/Styling2ndGen.md) 
guidelines

## How Can This Be Tested/Reviewed?

Compare the [dev storybook](https://storybook.bloom.exygy.dev/?path=/story/actions-expandable-content--standard) to the [branch one](https://deploy-preview-2829--dev-storybook-bloom.netlify.app/?path=/story/actions-expandable-content--standard) and verify they are stylistically identical.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
